### ISSUE TITLE
Is it possible to unload icudt55l.dat?

### ISSUE THREAD

**Issue by grayskripko** [gagolews/stringi#241]
When I change .libPaths() to test my package installation process it loads stringi package. After that, I can use unload, detach and dyn.unload to release test libraries resources. But I can not delete stringi because of loaded icudt55l.dat. I don't want to restart R session because it does not make sense in a script.

**Comment by gagolews** [comment]
icudt is being automatically loaded by the icu libraries (icui18n, etc.)

just out of curiosity: why would you want to unload this package on run-time?

BTW, are you working in RStudio? RStudion requires stringi on run-time

**Comment by grayskripko** [comment]
yes, I'm working on RStudio. 
As I said, I test my [package](https://github.com/cargomoose/ggraptR) installation. The package depends on many other packages so sometimes I can miss something in DESCRIPTION. For testing purpose, I created an autotesting script. It creates a new empty directory where I reproduce the installation process on naked R. After this process I want to turn all back. I can change .libPaths easily but I can not delete test directory because it contains icudt55l.dat which is used. 
Finally, I found a workaround. I just don't test stringi. I load it in the beginning of the script using library(). After that devtools::install_github does not require it
